src/EnergyPlus/HVACManager.cc

```diff
@@ -2394,22 +2267,22 @@ void ReportInfiltrations(EnergyPlusData &state)
                 thisInfiltration.MCpI_temp * (thisZoneHB.MAT - thisZone.OutDryBulbTemp) * TimeStepSysSec * ADSCorrectionFactor;
             thisInfiltration.InfilHeatGain = 0.0;
 
-        } else if (thisZoneHB.MAT <= thisZone.OutDryBulbTemp) {
+        } else {
 
             thisInfiltration.InfilHeatGain =
                 thisInfiltration.MCpI_temp * (thisZone.OutDryBulbTemp - thisZoneHB.MAT) * TimeStepSysSec * ADSCorrectionFactor;
             thisInfiltration.InfilHeatLoss = 0.0;
         }
 
         // Report infiltration latent gains and losses
-        H2OHtOfVap = PsyHgAirFnWTdb(thisZoneHB.ZoneAirHumRat, thisZoneHB.MAT);
+        H2OHtOfVap = Psychrometrics::PsyHgAirFnWTdb(thisZoneHB.ZoneAirHumRat, thisZoneHB.MAT);
         if (thisZoneHB.ZoneAirHumRat > state.dataEnvrn->OutHumRat) {
 
             thisInfiltration.InfilLatentLoss =
                 thisInfiltration.InfilMdot * (thisZoneHB.ZoneAirHumRat - state.dataEnvrn->OutHumRat) * H2OHtOfVap * TimeStepSysSec;
             thisInfiltration.InfilLatentGain = 0.0;
 
-        } else if (thisZoneHB.ZoneAirHumRat <= state.dataEnvrn->OutHumRat) {
+        } else {
```

Comment:
[src/EnergyPlus/HVACManager.cc:2412]:(style),[multiCondition],Expression is always true because 'else if' condition is opposite to previous condition at line 2406.